operators/src/pro/meta/quota.rs

```diff
@@ -34,3 +36,10 @@ impl QuotaTracking {
         let _ = self.quota_sender.send(self.computation); // ignore the Result because the quota receiver should never close the receiving end of the channel
     }
 }
+
+#[async_trait]
+pub trait QuotaCheck {
+    async fn check_quota(&self) -> Result<bool>;
```

Comment:
It should be more verbose what it is doing. For instance, `has_quota_available` would be an option.

Maybe you can add some doc-comments as well.

Reply:
I don't see in in this change, maybe you added the wrong one?

Reply:
a845e97cc0954aa90fb5c2acd53d1cda62c98a5f